### PR TITLE
examples : add RS-422 sample

### PR DESCRIPTION
**Pull request by shino-ixy**
Add Example for NextStep, For Spresense RS-422/485 AddOn Board.
This Program can select RD/DE pin from I2C_DIN/I2C_DOUT, EMMC2/EMMC3,
and use UART2 (115200bps).

https://nextstep.official.ec/items/38434310
It's work look line here

https://qiita.com/ixy-shino/items/e463c03d56b31ac5d0a7
I'm so happy you will review these codes in your coffee time.

### CODE REVIEW

**Comment by shino-ixy** [comment]
Please wait a moment, remove merge.

**Comment by shino-ixy** [comment]
Thank you for waiting.
I've ready to merge for the pull request.

**Comment by SPRESENSE** [comment]
@shino-ixy san, could you please add sdk/configs/examples/rs422/README.txt?
Thanks.

**Comment by shino-ixy** [comment]
OK! I've just written README.txt, Is this content sufficient?

**Comment by SPRESENSE** [comment]
@shino-ixy san
README.txt you added is very helpful. Thank you.

And, please refer to [README.txt](https://github.com/sonydevworld/spresense/blob/develop/sdk/configs/examples/alarm/README.txt), and add a simple README.txt under sdk/configs/examples/rs422 directory.

**Comment by shino-ixy** [comment]
@SPRESENSE Thank you for checking, I've just added README in configs.

File: sdk/configs/examples/rs422/README.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This configuration contains required options to use gyro example.
```
Comment (SPRESENSE) [inline sdk/configs/examples/rs422/README.txt:1]:
```suggestion
This configuration contains required options to use rs422 example.
```

File: sdk/configs/examples/rs422/README.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+[Source path]
+examples/gyro
```
Comment (SPRESENSE) [inline sdk/configs/examples/rs422/README.txt:4]:
```suggestion
examples/rs422
```

**Review by SPRESENSE — CHANGES_REQUESTED** [review]
@shino-ixy , please modify an example name as I suggested.

**Review by SPRESENSE — APPROVED** [review]
@shino-ixy san, thank you!!

**Pull request merged**